mappings/net/minecraft/realms/DisconnectedRealmsScreen.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4899 net/minecraft/realms/DisconnectedRealmsScreen
+	FIELD field_22807 titl Ljava/lang/String;
```

Comment:
```suggestion
	FIELD field_22807 title Ljava/lang/String;
```

Reply:
right, there was a conflict from screen title and i forgot to fix it :trollface: